admin/js/boldgrid-backup-admin-folder-exclude.js

```diff
@@ -352,14 +352,16 @@ BoldGrid.FolderExclude = function( $ ) {
 	 *
 	 * @since 1.6.0
 	 */
-	self.toggleConfig = function() {
-		var type = $type.filter( ':checked' ).val(),
-			$miscInfo = $( '#folder_misc_info' );
+	self.toggleConfig = function( typeInput ) {
+		var type = $( typeInput )
+			.filter( ':checked' )
+			.val();
+		var $miscInfo = $( 'div#folder_misc_info' );
```

Comment:
We can put everything in one `var` statement - https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#declaring-variables-with-var